python/cudf_polars/tests/expressions/test_stringfunction.py

```diff
@@ -481,6 +481,72 @@ def test_string_join(ldf, ignore_nulls, delimiter):
     assert_gpu_result_equal(q)
 
 
+@pytest.mark.parametrize(
+    "fill",
+    [
+        0,
+        1,
+        2,
+        5,
+        999,
+        -1,
+        None,
+    ],
+)
+@pytest.mark.parametrize(
+    "input_strings",
+    [
+        ["1", "0"],
+        ["123", "45"],
+        ["", "0"],
+        ["-1", "+2"],
+        ["abc", "def"],
+    ],
+)
+def test_string_zfill(fill, input_strings):
```

Comment:
Can you run these tests with polars 1.29? You _might_ need to version guard these tests ie `if POLARS_VERSION_LT_130`